packages/labs/ssr-react/README.md

```diff
@@ -0,0 +1,72 @@
+
+```js
+import React from 'react';
+import {patchCreateElement} from `@lit-labs/ssr-react`;
+
+const patchedCreateElement = patchCreateElement(React.creatElement);
+```
+
+When compiling your source code's JSX to js, you can instruct your tool to use the `patchedCreateElement` function instead of React's original `createElement` by
+
+#### Adding a JSX pragma
+
+```js
+/** @jsx patchedCreateElement */
+```
+
+#### Or, configuring your compiler
+
+- For Babel: specify the [`pragma`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragma) option for `@babel/plugin-transform-react-jsx`.
+- For TypeScript: specify the [`jsxFactory`](https://www.typescriptlang.org/tsconfig#jsxFactory) option in `tsconfig.json`.
+
+Note: Both of the solutions above require the function `patchedCreateElement` (or whatever specifier you choose for the patched function) already be present in scope of the file with JSX to transform. Each `jsx`/`tsx` file will need to include the code snippet at the top of this section. Therefore, it is recommended to use the `@lit-labs/ssr-react/enable-lit-ssr.js` module mentioned below.
+
+### Using the Runtime JSX Transform
+
+If your project is using the [runtime JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html), this package is configured to be able to serve as the JSX import source whose JSX functions have been patched to enable Lit SSR.
+
+- For Babel: specify the [`importSource`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#importsource) option in `@babel/plugin-transform-react-jsx` as `@lit-labs/ssr-react`.
+- For TypeScript: specify the [`jsxImportSource`](https://www.typescriptlang.org/tsconfig#jsxImportSource) option in `tsconfig.json` as `@lit-labs/ssr-react`.
+
+### Monkey patching `React.createElement`
+
+This package provides a module that, when imported, has a side-effect of monkey patching `React.createElement` to be the patched version. This can be imported at the entry point of the application before `React` is imported.
+
+```js
+import '@lit-labs/ssr-react/enable-lit-ssr.js';
+```
+
+The advantage of this approach is that it does not require modification to any other files containing JSX. It will also work for external components that have been pre-compiled with the classic JSX transform or React components made from Lit components using `@lit-labs/react`.
+
+### Note on Enabling Hydration
+
+Both the Runtime JSX Transform replacement and using `@lit-labs/ssr-react/enable-lit-ssr.js` have conditional exports configured such that when imported for a server environment they enhance React's original element creation to deeply render Lit components, and when imported for the browser will bring in `lit/experimental-hydrate-support.js` and provide pass-throughs for React's original element creation.
```

Comment:
In theory you might want to use Lit SSR with React but not actually need hydration. In that case always including experimental hydration may not be desirable. Thoughts on that?

Reply:
I feel like vast majority of use case would want hydration so making that easier sounds better to me. I'm also thinking users who want no hydration for their React app will have a different entry point for their client JS, if any, which can avoid the auto imports.